### PR TITLE
Forgot/Recover Password

### PR DESCRIPTION
**Pull request by pirvanrobert**
gata BAAAAAAAAAAAAAAAAAAAAAAAAAAAAA

### CODE REVIEW

File: src/App.jsx [hunk 1]
```diff
@@ -10,10 +18,13 @@ function App() {
 		<div>
 			<Routes>
 				<Route element={<TasksPage />} path="/" />
+				<Route element={<RecoverPasswordWrapper />} path="/recover" />
 
 				<Route element={<TestPage />} path="/test" />
 				<Route element={<SingInPage />} path="/login" />
 				<Route element={<RegisterPage />} path="/register" />
+				<Route element={<ResetPassword />} path="/reset" />
+				<Route element={<RecoverPassword />} path="/recoverPass" />
```
Comment (shapovalov-sasha) [inline src/App.jsx:27]:
```suggestion
				<Route element={<RecoverPassword />} path="/recover-password" />
```

File: src/pages/reset-password/ResetPassword.jsx [hunk 2]
```diff
@@ -0,0 +1,18 @@
+import React from "react";
+
+import { Box } from "@mui/material";
```
Comment (shapovalov-sasha) [inline src/pages/reset-password/ResetPassword.jsx:3]:
```suggestion
import { Box, Typography } from "@mui/material";
```

**Comment by Adrian-Edu** [comment]
aaaa

File: src/pages/reset-password/ResetPassword.jsx [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+import Typography from "@mui/material/Typography";
```
Comment (shapovalov-sasha) [inline src/pages/reset-password/ResetPassword.jsx:4]:
```suggestion

```

**Review by shapovalov-sasha — CHANGES_REQUESTED** [review]

**Comment by Adrian-Edu** [comment]
e ok

**Review by shapovalov-sasha — APPROVED** [review]

**Review by Eladie — APPROVED** [review]

**Pull request merged**